src/Akka.Streams.Kafka.Tests/Integration/CommittableSourceIntegrationTests.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Streams.Dsl;
+using Akka.Streams.Kafka.Messages;
+using Akka.Streams.Kafka.Settings;
+using Akka.Streams.TestKit;
+using Confluent.Kafka;
+using Confluent.Kafka.Serialization;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Streams.Kafka.Tests.Integration
+{
+    public class CommittableSourceIntegrationTests : Akka.TestKit.Xunit2.TestKit
+    {
+        private const string KafkaUrl = "localhost:9092";
+
+        private const string InitialMsg = "initial msg in topic, required to create the topic before any consumer subscribes to it";
+
+        private readonly ActorMaterializer _materializer;
+
+        public CommittableSourceIntegrationTests(ITestOutputHelper output) 
+            : base(ConfigurationFactory.FromResource<ConsumerSettings<object, object>>("Akka.Streams.Kafka.reference.conf"), null, output)
+        {
+            _materializer = Sys.Materializer();
+        }
+
+        private string Uuid { get; } = Guid.NewGuid().ToString();
+
+        private string CreateTopic(int number) => $"topic-{number}-{Uuid}";
+        private string CreateGroup(int number) => $"group-{number}-{Uuid}";
+
+        private ProducerSettings<Null, string> ProducerSettings =>
+            ProducerSettings<Null, string>.Create(Sys, null, new StringSerializer(Encoding.UTF8))
+                .WithBootstrapServers(KafkaUrl);
+
+        private async Task GivenInitializedTopic(string topic)
+        {
+            var producer = ProducerSettings.CreateKafkaProducer();
+            await producer.ProduceAsync(topic, null, InitialMsg, 0);
+            producer.Dispose();
+        }
+
+        private ConsumerSettings<Null, string> CreateConsumerSettings(string group)
+        {
+            return ConsumerSettings<Null, string>.Create(Sys, null, new StringDeserializer(Encoding.UTF8))
+                .WithBootstrapServers(KafkaUrl)
+                .WithProperty("auto.offset.reset", "earliest")
```

Comment:
This doesn't have to happen now, but my idea was to extract producer/consumer config values from documentation and expose them as `Property<T>(string config, T value)`. This way you can have type-safe properties, that could be suggested by intellisense.